host.yaml

```diff
@@ -102,6 +102,8 @@ sm_options:
   rom_start: true
 factorio_options:
   executable: "factorio\\bin\\x64\\factorio"
+  # by default, no settings are loaded if this file does not exist.  If this file does exist, then it will be used.
+  server_settings: "factorio\\data\\server-settings.json"
```

Comment:
Does forward slash work on windows as well? because then i would prefer it to be forward slash for us linux and mac users.

And secondly is there a reason to set it to the that path and not have it use default by default? If not then i think it should default to None in Utils.py and be a changed to a comment in host.yaml.

Reply:
I am guessing backslash doesn't work in linux / mac environments.  Forward slashes do work in windows environments, in which case, by extension, the executable path should be converted to forward slashes as well.